src/OrchardCore.Modules/OrchardCore.AdminMenu/AdminNodes/PlaceholderAdminNodeDriver.cs

```diff
@@ -22,16 +34,59 @@ public override IDisplayResult Edit(PlaceholderAdminNode treeNode)
             {
                 model.LinkText = treeNode.LinkText;
                 model.IconClass = treeNode.IconClass;
+                model.SelectedItems = new List<VueMultiselectItemViewModel>();
+                model.AllItems = new List<VueMultiselectItemViewModel>();
+
+                var nameList = new List<string>();
+                foreach (var permission in treeNode.Permissions)
+                {
+                    nameList.Add(permission.Name); 
+
+                    model.SelectedItems.Add(new VueMultiselectItemViewModel
+                    {
+                        Id = permission.Name,
+                        DisplayText = $"{permission.Name} - {permission.Description}"
+                    });
+                }
+                model.PermissionIds = string.Join(",", nameList);
+                
+                foreach (var permission in _permissionService.GetInstalledPermissionsAsync().Result)
```

Comment:
For info, if it fails exceptions are better managed by using `.GetAwaiter().GetResult()` in place of `.Result`

But here better to override `DisplayAsync()` and then use a regular async / await